examples/text-classification/create_dataset.py

```diff
@@ -64,6 +64,8 @@ def create_dataset(folder, input_file_name, db_batch_size=None, create_images=Fa
             sample = np.ones(FEATURE_LEN) # one by default (i.e. 'other' character)
             count = 0
             for field in row['fields']:
+                if count > FEATURE_LEN-1:
+                    break
```

Comment:
this looks OK to me. Alternatively we could do:

```python
for char in field.lower():
    if count < FEATURE_LEN:
        if char in cdict:
            sample[count] = cdict[char]
    else:
        break
    count += 1
```

so we'd have only one condition on the length of the `sample` vector

```